src/main/java/com/minecolonies/coremod/client/render/RenderBarbarianMob.java

```diff
@@ -0,0 +1,45 @@
+package com.minecolonies.coremod.client.render;
+
+import com.minecolonies.coremod.entity.ai.mobs.EntityBarbarian;
+import net.minecraft.client.model.ModelBiped;
+import net.minecraft.client.renderer.entity.Render;
+import net.minecraft.client.renderer.entity.RenderLiving;
+import net.minecraft.client.renderer.entity.RenderManager;
+import net.minecraft.client.renderer.entity.layers.LayerHeldItem;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.fml.client.registry.IRenderFactory;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Created by Asher on 5/6/17.
+ */
+public class RenderBarbarianMob extends RenderLiving<EntityBarbarian>
+{
+    final private ResourceLocation mobTexture = new ResourceLocation("minecolonies:textures/entity/barbarian1.png");
+
+    public static final Factory FACTORY = new Factory();
+
+    public RenderBarbarianMob(final RenderManager renderManagerIn)
+    {
+        super(renderManagerIn, new ModelBiped(), 0.5F);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.5F to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)